marklogic-data-hub-spark-connector/gradle.properties

```diff
@@ -1,3 +1,6 @@
 junitJupiterVersion=5.4.0
 junitPlatformVersion=1.4.0
 sparkVersion=2.4.5
+#Properties to be used for testing locally using gradle
```

Comment:
Toss a line above this just to space it out from the version numbers. 

Then - these properties are going to be used for testing in any environment, right?

Reply:
Yes, We can use these in any environment. Should I create a new gradle-dhs.properties file?

Reply:
Keep them in here - they have to exist in gradle.properties so that you can reference them in build.gradle. The only reason for a "gradle-env.properties" file is if there's an "env" that we want to configure properties for. That isn't the case and likely won't be for this project. We'll always use e.g. "-P" to pass in env-specific properties for things like a Jenkins environment.